app/page.tsx

```diff
@@ -43,10 +45,33 @@ const ITEMS_PER_PAGE = 12;
 const SEARCH_DEBOUNCE_DELAY = 600;
 const DEFAULT_FILTER_OPTION = filterOptions[0];
 
+type CategoryOptions =
+  | ""
+  | "politics"
+  | "crypto"
+  | "technology"
+  | "sports"
+  | "international"
+  | "business"
+  | "entertainment"
+  | "economy";
+
+const categories: CategoryOptions[] = [
+  "technology",
+  "crypto",
+  "politics",
+  "economy",
+  "business",
+  "international",
+  "sports",
+  "entertainment",
+];
+
 export default function HomePage() {
   const router = useRouter();
   const showClientUI = useShowClientUI();
   const [isPopoverOpen, setPopoverOpen] = useState(false);
+  const [category, setCategory] = useState<CategoryOptions>("");
```

Comment:
If you decide to use the enum, this change may be needed:

```suggestion
  const [category, setCategory] = useState< CATEGORIES>();
```